runner/src/scenario/ceramic/model_instance.rs

```diff
@@ -180,9 +188,24 @@ impl CeramicModelInstanceTestUser {
                     &config.redis_cli,
                     lead_user,
                     Some(goose::get_worker_id().to_string()),
+                    false,
                 )
                 .await?
             }
+            crate::scenario::ceramic::ReuseType::LeadWorkerSubscriber => {
+                let (small, large) = Self::generate_list_models(
+                    user,
+                    &config.admin_cli,
+                    &config.redis_cli,
+                    global_leader,
+                    None,
+                    true,
+                )
+                .await?;
+                Self::ensure_model_exists(user, &config.user_cli, &small).await?;
+                Self::ensure_model_exists(user, &config.user_cli, &large).await?;
+                (small, large)
+            }
         };
 
         if lead_user {
```

Comment:
Because of this `if`-block, one user per worker will call the model subscription API, which means each nod will already be subscribed to the models. This means that we shouldn't need to pass the `should_subscribe` flag as you're doing below.